core/vtk/ttkPlanarGraphLayout/ttkMergeTreeVisualization.h

```diff
@@ -1300,6 +1328,15 @@ class ttkMergeTreeVisualization : public ttk::MergeTreeVisualization {
                 = (toAddT == 1
                      ? (isPDSadMax or nodeIsRoot ? locMax : saddle1)
                      : (not isPDSadMax or nodeIsRoot ? locMin : saddle2));
+              if(embeddedDiagram) {
+                bool nodeSup = trees[i]->getValue<dataType>(node)
+                               > trees[i]->getValue<dataType>(nodeOrigin);
+                // TODO is this correct?
```

Comment:
Is this correct then? 
(if so, thanks for removing the comment)

Reply:
It seems to!